### DISCUSSION TITLE
Redirect error, not sure if it's Mastodon or something else?

### DISCUSSION THREAD

**Discussion by chrisferrantelli** [mastodon/mastodon#20140] [Q&A]
When I get to the step with certbot and run the command `certbot --nginx -d example.com` , I'd type my domain in my browser and it would bring me to the screen with the Elephant on the computer as the docs state. 

However when I complete everything on the `Setting up systemd services` part I just get an error in my browsers about there being redirect errors. Now I feel this is a Mastodon thing and there is something I'm missing because I really feel that it should have said the same thing instead of showing the elephant previously. 

I know it's not my records because I triple checked and everything's correct.

Not sure if it helps tho but my domain is from Namecheap and I'm pointing it to my VPS on Contabo.

**Comment by tuetenk0pp** [comment]
Are you sure you set your domain in the nginx config and .env.production files?

**Reply by chrisferrantelli** [reply]
Found out the issue after an hour or 2 of more research ,  for some reason certbot makes another `server` block for the 443 port. I removed that one for and kept the one that was already there and worked no issues.
